src/kakarot/accounts/contract/contract_account.cairo

```diff
@@ -61,22 +61,22 @@ func bytecode_len{
 }
 
 // @notice Store a key-value pair.
-// @param key The bytes32 storage key.
+// @param key The storage key, which is hash_felts(cast(Uint256, felt*)) of the Uint256 storage key.
```

Comment:
we don't use sn_keccak anymore on the name of the storage slot?

Reply:
well actually this is an _internal_ storage key, not the same storage key as the one used by Starknet. I did this to go faster after I got no response for a `get_storage_var_address` implementation in cairo.

I can update now that everything else work, it'll be better

Reply:
I've updated to use as `dict` key the actual `storage_addr` of the storage var of the underlying ContractAccount. Tbh I don't understand the benefit of this change though it looks reasonnable, logically speaking.

Reply:
But I was saying, we don't use the storage_ storage variable anymore right?

We could delete the line where we write:
@storage_var
storage_ ...